content/How_To/GUI/MRTK.md

```diff
@@ -0,0 +1,137 @@
+
+The `NearMenu` also has a pin button, that when pressed, will freeze the menu in space. The menu can also be grabbed around the edges to move it (it comes with an attached `SixDofDragBehavior`), at which point it becomes pinned in order to maintain its new position. 
+
+#### Hand Menu
+The `HandMenu` is a `HolographicMenu` that is anchored to the user's hand using a `HandConstraintBehavior`.
+
+Its default position lies on the outside of the user's left hand, such that when their palm is up, the menu is facing them and easily accessible by their right hand.
+
+The menu can be customized to appear by default on the right hand or either hand, and can also be anchored to a different quadrant of the hand. Only one menu will be visible at a time though, even if both hands are up. More information on this can be found in the `HandConstraintBehavior`'s section [here](divingDeeper/gui/mrtk#hand-constraint-behavior).
+
+#### Slider 3D
+The `Slider3D` is a customizable slider with a 3D presence. The default range of the slider is from `0` to `1`, though that can be customized, along with the step count and precision.
+
+The `Slider3D` can be created with a backplate, which is determined on creation. After creation, the `minimum` and `maximum` values can be adjusted, as well as the `step` size. 
+
+See it in action here:
+<Playground id="#2PKNJB#7" title="Slider Color Changer" description="A basic example showing how 3D sliders can affect their surroundings."/>
+
+## Behaviors and Gizmos
+Some behaviors and Gizmos are also shared with MRTK components. In MRTK terminology, these are usually referred to as Solvers, Bounds Control, or Object Manipulators. Behaviors and Gizmos are attached to `Nodes`, `TransformNodes`, and `Meshes` in order to provide additional functionality.
+
+More information on `Gizmos` can be found [here](divingDeeper/mesh/gizmo), while behaviors are covered in more detail [here](divingDeeper/behaviors/meshBehaviors).
+
+#### SixDof Drag Behavior
+The `SixDofDragBehavior`, also known as MRTK's `Object Manipulator` component. By creating and attaching it to an object in the scene, users are able to grab, move, and rotate the object. Interacting with a ray-cast input allows the object to be moved in 3-dimansions, while interacting with near interaction adds the ability to rotate the object in-place.
+```javascript
+var sixDofDragBehavior = new BABYLON.SixDofDragBehavior();
+```
+To attach a `SixDofDragBehavior` to a mesh in the scene, simply call the mesh's `addBehavior` function:
+```javascript
+mesh.addBehavior(sixDofDragBehavior);
+```
+#### Follow Behavior
+The `FollowBehavior` tries to maintain a set distance between the `TransformNode` it's attached to, and the `followedCamera` target (defaults to main camera).
+
+There are many aspects to this behavior that can be customized, these are the core properties:

```

Comment:
Suggestion:
There are many aspects to this behavior that can be customized. These are the core properties: